src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -0,0 +1,507 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+"""
+Panda Power Converter
+"""
+import re
+from functools import lru_cache
+from typing import Dict, Mapping, Optional, Union
+
+import numpy as np
+import pandas as pd
+from power_grid_model import Branch3Side, BranchSide, LoadGenType, initialize_array
+from power_grid_model.data_types import Dataset
+
+from power_grid_model_io.converters.base_converter import BaseConverter
+from power_grid_model_io.data_types import ExtraInfoLookup
+from power_grid_model_io.filters import get_winding
+
+StdTypes = Mapping[str, Mapping[str, Mapping[str, Union[float, int, str]]]]
+PandasData = Mapping[str, pd.DataFrame]
```

Comment:
I see that it is often called Pandas data, do we want this? It is indeed just a Pandas dataframe, but don't we want to call it PandaPowerData here? Since this module will only be used for pandapower and not for other Pandas DataFrames

Reply:
It's not 'just' pandas data, it is a dictionary of str to DataFrame. But indeed, PandaPowerData may be better as it is only used in this file.